test/assignment5/multiple_packets.txt

```diff
@@ -0,0 +1,4 @@
+first packet
+01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901 
+third packet
```

Comment:
@DeekshithPatil I'd like to avoid adding another file... either just use newline in the string you send (ideally) or a heredoc in the shell script.